packages/hash/frontend/src/blocks/page/MentionView/MentionDisplay.tsx

```diff
@@ -30,8 +31,8 @@ export const MentionDisplay: FunctionComponent<MentionDisplayProps> = ({
         } else {
           // Once the query loads, either display the found name, or display "Unknown User" if the user doesn't exist in the users array
           userName =
-            users.find((item) => item.entityId === entityId)?.preferredName ??
-            "Unknown User";
+            users.find((user) => user.entityEditionId.baseId === entityId)
+              ?.preferredName ?? "Unknown User";
         }
 
         return {
```

Comment:
Potentially worth fixing as part of this PR, I think a couple lines down the `href` includes a `%` which throws a run-time error.
<img width="1335" alt="image" src="https://user-images.githubusercontent.com/42802102/203803802-e2e7e36a-65aa-4dd0-84d3-370f5ea39e87.png">

Reply:
Good catch, thanks!